src/spryker/harness.yml

```diff
@@ -0,0 +1,78 @@
+---
+harness('inviqa/spryker'):
+  description: A docker based development environment for Spryker
+  require:
+    services:
+      - proxy
+    confd:
+      - harness:/
+---
+attributes:
+  app:
+    services:
+      - chrome
+      - postgres
+      - redis
+      - elasticsearch
+      - rabbitmq
+      - jenkins
+      - jenkins-slave
+  spryker:
+    salt: 2tkqCCRKy5rT4wMVz8KTCh8r3sJGkL5v
+    demoshop-url: https://github.com/spryker-shop/b2c-demo-shop.git
```

Comment:
Is it possible to override this with the b2b demoshop if needed at the time of creating a project?

Reply:
That is my plan, not sure yet how to do that but keeping this value configurable in yml is the first step